x-pack/plugin/src/test/resources/rest-api-spec/test/async_search/10_basic.yml

```diff
@@ -6,20 +6,32 @@
         body:
           settings:
             number_of_shards: "2"
+          mappings:
```

Comment:
Instead of making these changes it *should* be possible for me to add an index template that automatically does the conversion.